src/scancode/cli.py

```diff
@@ -382,6 +382,9 @@ def scan_event(item):
                     return ''
                 if item:
                     _scan_success, _scanned_path = item
+                    if len(_scanned_path)>20:
+                    	current_filename, current_file_extension = os.path.splitext(_scanned_path)
```

Comment:
Please consider the use of the `commoncode.fileutils` functions instead of splitext